Makefile

```diff
@@ -752,9 +751,19 @@ devbox-grpc:
 		lib/multiplexer/test/ping.proto \
 		lib/web/envelope.proto
 
+	# proto files within the api module must be passed with the 'M' flag. This way
+	# protoc will generate files with the correct import paths in the case where
+	# the import path has a version suffix, e.g. github.com/gravitational/teleport/api/v8
+	$(eval API_IMPORT_PATH=$(shell go run build.assets/go-modules/print-api-import-path/main.go))
+
 	protoc -I=.:$$PROTO_INCLUDE \
 		--proto_path=api/client/proto \
-		--gogofast_out=plugins=grpc:api/client/proto \
+		--gogofast_out=plugins=grpc,\
+Mgithub.com/gravitational/teleport/api/types/types.proto=$$API_IMPORT_PATH/types,\
```

Comment:
This is kind of a pain, as proto generation will break for every new file. I wish we could get around it some other way, but can't think of any alternatives right now (apart from the obvious "move the protos out of the Go module").

Reply:
What do you mean by proto generation breaking down for new files?

Reply:
Nothing has really changed here though, right? We've always listed all the .proto files explicitly and adding a new one requires updates.

Reply:
I meant that we have to add new files manually to the list, but yes, nothing much changed.